internal/component/discovery/process/cgroup_test.go

```diff
@@ -0,0 +1,53 @@
+//go:build linux
+
+package process
+
+import (
+	"bytes"
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestGenericCGroupMatching(t *testing.T) {
```

Comment:
Can we add a test for the update regex functionality? Looks like its coming along nicely!

Reply:
I pushed couple of unit tests. Could you check them and let me know if that is what you mean? Cheers